source/includes/_oy-tutorial.md

```diff
@@ -187,3 +187,74 @@ You can enable or disable notification settings for disbursement products if you
 2. The email field:  allows the user to add, remove, and edit email notification receivers in this column, which can hold up to three email addresses.
 3. Click ‘Save Changes’
 
+## Multi Account Management
+
+Multi Account Management (Subsidiaries) is a feature that can help you handle complex relations between entities in your business in the Oy! Dashboard. Please contact our Sales or Account Management if you’re interested to use this feature.  
+With this feature you will be able to :
+
+1. Have a detailed report on transactions performed by all of subsidiary account
+2. Distribute balance between the parent account to subsidiaries account
+3. Use subsidiary balance to disburse funds
+
+### **Registration & Setup**
+
+**Prerequisites**
+
+Register an account on the [OY! dashboard](https://business.oyindonesia.com/register?)
+Activate your account through the activation link sent via email
+Upgrade your account
+Upgrade request is approved
+Contact our sales or account manager representatives to enable the feature
+
+### **How To Use**
+
+To properly used Multi Account Management, we required several accounts to be linked together. Each account can be used independently, and each account needs to pass the KYB to be used fully.
+
+**A. Link to Subsidiary Account**
+
+1. **For Parent Account**
+
+To add a new subsidiary as a parent account please 1) Log in to Dashboard with parent credential 2) Go to Subsidiaries → List of Subsidiaries
+
+![As Parent - Subs - 1. Initial.png](../images/As_Parent_Subs_1_Initial(2).png)
+
+1. Click the “+ Add Subsidiary”
+2. Fill in the username for the subsidiary you wished to connect. You can find your username from the menu Accounts → User Profile in the Username section
+3. Click Add and then if the account is found in the system you can confirm by clicking Connect in the confirmation window
+
+![Image: As Parent - Subs - 6. Add Subsidiary - Type Username](../images/As_Parent_Subs_6_Add_Subsidiary_Type_Username.png)![Image: As Parent - Subs - 7a. Existing Username - Not Connected with any parent](../images/As_Parent_Subs_Existing_Username_Not Connected_with_any_parent.png)After confirming, the request will be sent to the respective account to be accepted
```

Comment:
![image](https://user-images.githubusercontent.com/87056482/136758426-8de1e7d2-8356-4112-a6fa-00d416a5d6c0.png)
`![Image: As Parent - Subs - 7a. Existing Username - Not Connected with any parent](../images/As_Parent_Subs_Existing_Username_Not Connected_with_any_parent.png)` nggk kerender, kayaknya kurang enter di awal sama akhirnya

Reply:
ini udah gw kasih enter juga ga bisa ya kalo di githubnya
...tapi kalo dibuka dilocal aman kok ke render

Reply:
udah fixya ternyata nama imagenya ada yang masih pake spasi..bisa dicek di commit latest